.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.pythonPath": "/home/patrick/miniconda3/envs/human-detection/bin/python",
```

Comment:
This path is specific to you, and are these VS code settings important to all of us?